superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterControls.tsx

```diff
@@ -103,7 +103,7 @@ const FilterControls: FC<FilterControlsProps> = ({
       return (
         // Empty text node is to ensure there's always an element preceding
         // the OutPortal, otherwise react-reverse-portal crashes
-        <React.Fragment key={id}>
+        <React.Fragment>
           {'' /* eslint-disable-line react/jsx-curly-brace-presence */}
           <OutPortal node={portalNodes[index]} inView />
         </React.Fragment>
```

Comment:
To prevent `Prefer fragment shorthand over React.Fragment` warning.
```suggestion
        <>
          {'' /* eslint-disable-line react/jsx-curly-brace-presence */}
          <OutPortal node={portalNodes[index]} inView />
        </>
```

Reply:
Updated PR with `key={index}` so this comment is no longer relevant